lib/std/os/bits/darwin.zig

```diff
@@ -1554,3 +1554,186 @@ pub const rlimit = extern struct {
 pub const SHUT_RD = 0;
 pub const SHUT_WR = 1;
 pub const SHUT_RDWR = 2;
+
+// Term
+pub const VEOF = 0;
```

Comment:
Would a request to add comments from the matching libc header file be an overkill? See [here](https://github.com/ziglang/zig/blob/master/lib/libc/include/x86_64-macos-gnu/sys/termios.h#L76) for comments.

Reply:
Done!